dpctl-capi/include/dpctl_sycl_device_interface.h

```diff
@@ -126,7 +126,7 @@ bool DPCTLDevice_IsHost(__dpctl_keep const DPCTLSyclDeviceRef DRef);
  */
 DPCTL_API
 DPCTLSyclBackendType
-DPCTLDevice_GetBackend(__dpctl_keep const DPCTLSyclDeviceRef DRef);
+DPCTLDevice_GetBackendVersion(__dpctl_keep const DPCTLSyclDeviceRef DRef);
```

Comment:
This function does not need renaming. The SYCL function is called `get_backend` too.

Reply:
@1e-to can make this change? I will like to have this change in 0.7.0.

Reply:
Oops missed your change. All set.